errors/errors.go

```diff
@@ -34,4 +34,8 @@ var (
 	ErrConfigNotFound          = errors.New("cli: config with the given name does not exist")
 	ErrNoURLProvided           = errors.New("cli: no URL provided in argument or via config. see 'zot config -h'")
 	ErrIllegalConfigKey        = errors.New("cli: given config key is not allowed")
+	ErrNotSupported            = errors.New("search: squashfs image scanning not supported")
+	ErrJSONNotFound            = errors.New("repository: index.json not found")
```

Comment:
Where was this needed?

Reply:
I used this errors message while checking valid image format to scan in cve.go